index.js

```diff
@@ -4,7 +4,34 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    if (process.argv.length <= 2) {
+        return "No operation provided...";
+
+    } else if (process.argv[2] === "modulo") {
+          return "Invalid operation: modulo";
+
+    } else if (process.argv[2] === "divide") {
+        return "Invalid operation: divide";
+
```

Comment:
These two cases will only catch some (not all) unrecognized operators. What if I ran `node run.js exponentiate 2 7`?  Try making a single conditional that will catch any/all unrecognized operators.

Reply:
Thanks for the feedback! I changed the file in order to refactor with your tips in mind.

Reply:
Can I send another pull request for this function if my first pull request has not been confirmed. Are we always sending the pull request to the"pre-forked" repo?

If you accept a "pull request" does it permanently change your main /joinpursuit/ directory? Does that mean we are only sending Pull Requests to practice but not to change the main /joinpursuit? repo?

Reply:
@myrasmith Just refactored this lab without JSON. What do you think?